tests/conftest.py

```diff
@@ -131,6 +131,7 @@ def db(webapp):
         models.db.session.close()
         models.db.drop_all()
         models.db.create_all()
+        yield models.db
```

Comment:
Can remove it?

Reply:
No, it's need for pytest-flask-sqlalchemy